subiquitycore/file_util.py

```diff
@@ -21,14 +21,27 @@
 
 import yaml
 
-_DEF_PERMS = 0o640
+_DEF_PERMS_FILE = 0o640
 _DEF_GROUP = 'adm'
 
 
+def set_log_perms(target, *, isdir=True, group_write=False, mode=None):
+    if os.getuid() != 0:
+        return
```

Comment:
I'm going to go with the warning level.  Normally it should only be seen in the dry-run case, but if it happens in a real environment it may be interesting.